openedx/adg/lms/webinars/helpers.py

```diff
@@ -137,3 +244,65 @@ def remove_emails_duplicate_in_other_list(email_list, reference_email_list):
         emails (list): list of remaining emails
     """
     return [email for email in email_list if email not in reference_email_list]
+
+
+def update_webinar_team_registrations(webinar_form):
+    """
+    Updates webinar team registrations if webinar presenter, cohosts or panelists are updated.
+
+    Args:
+        webinar_form (Form): Model form with updated data.
+
+    Returns:
+        None
+    """
+    from openedx.adg.lms.webinars.models import WebinarRegistration
+
+    webinar = webinar_form.instance
+    cleaned_data = webinar_form.cleaned_data
+    old_team = webinar.webinar_team()
+    new_team = set(chain(cleaned_data['co_hosts'], cleaned_data['panelists'], {cleaned_data['presenter']}))
+
+    newly_added_members = list(new_team - old_team)
+    removed_members = list(old_team - new_team)
+
+    if newly_added_members:
+        WebinarRegistration.create_team_registrations(newly_added_members, webinar)
+        schedule_webinar_reminders([user.email for user in newly_added_members], webinar)
+
+    if removed_members:
+        WebinarRegistration.remove_team_registrations(removed_members, webinar)
+
+        registrations = WebinarRegistration.objects.filter(
+            webinar=webinar, user__in=removed_members, is_registered=False
```

Comment:
Why are we adding the `is_registered=False` check? Shouldn't the reminders be cancelled for all removed members, irrespective of the fact whether they were registered or not?

Reply:
The reason is that user can register in webinar from the lms and in that case `is_registered` is True so we should not cancel reminders for him.